assets/quick-order-list.js

```diff
@@ -209,34 +209,42 @@ class QuickOrderList extends HTMLElement {
     }
     this.variantListInput = event.target;
     this.variantListInput.select()
-    this.variantListInput.addEventListener('keydown', (e) => {
-      if (e.key === 'Enter') {
-        e.preventDefault();
-        e.target.blur();
-        const currentIndex = this.allInputsArray.indexOf(e.target);
-
-        if (!e.shiftKey) {
-          const nextIndex = currentIndex + 1;
-          const nextVariant = this.allInputsArray[nextIndex] || this.allInputsArray[0];
-          nextVariant.select();
-        } else {
-          const previousIndex = currentIndex - 1;
-          const previousVariant = this.allInputsArray[previousIndex] || this.allInputsArray[this.allInputsArray.length - 1];
-          previousVariant.select();
+    if (this.allInputsArray.length !== 1) {
```

Comment:
is `this.allInputsArray.length <= 1` a better way to go?

also what do we expect to happen when a merchant presses `enter` on a single variant list? Should it do anything?

Reply:
> is this.allInputsArray.length <= 1 a better way to go?

My logic was that inside the Quick Order List there is always at least one input, the only case when there is no input would be when all variants are sold out but in this case we there is nothing to interact with. 

Is there other reasons to use `<= 1` instead of` !==1`? Or is it just a better principle in coding to avoid '!==" ?

>also what do we expect to happen when a merchant presses enter on a single variant list? Should it do anything?

We just want to remove focus from the input before updating started.